### ISSUE TITLE
gym looking for a library that is no longer present

### ISSUE THREAD

**Issue by photoplyr** [fastlane/fastlane#6131]
Our builds are failing now using gym and Xcode 8.0 set as Swift 2.3

Here is my command line: 
`gym --workspace "BusinessApp.xcworkspace" --scheme "BusinessApp" --clean --use_legacy_build_api`

Gym version:
gym 1.9.0

Error Log:

```
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1380:in `initialize': [31m[!] No such file or directory - /Applications/Xcode.app/Contents/Developer//Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphoneos/libswiftLocalAuthentication.dylib[0m (Errno::ENOENT)
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1380:in `open'
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1380:in `copy_file'
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:477:in `copy_file'
    from /Library/Ruby/Gems/2.0.0/gems/gym-1.9.0/lib/gym/xcodebuild_fixes/swift_fix.rb:30:in `block (2 levels) in swift_library_fix'
    from /Library/Ruby/Gems/2.0.0/gems/gym-1.9.0/lib/gym/xcodebuild_fixes/swift_fix.rb:27:in `each'
    from /Library/Ruby/Gems/2.0.0/gems/gym-1.9.0/lib/gym/xcodebuild_fixes/swift_fix.rb:27:in `block in swift_library_fix'
    from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/tmpdir.rb:88:in `mktmpdir'
    from /Library/Ruby/Gems/2.0.0/gems/gym-1.9.0/lib/gym/xcodebuild_fixes/swift_fix.rb:20:in `swift_library_fix'
    from /Library/Ruby/Gems/2.0.0/gems/gym-1.9.0/lib/gym/runner.rb:83:in `fix_package'
    from /Library/Ruby/Gems/2.0.0/gems/gym-1.9.0/lib/gym/runner.rb:21:in `run'
    from /Library/Ruby/Gems/2.0.0/gems/gym-1.9.0/lib/gym/manager.rb:10:in `work'
    from /Library/Ruby/Gems/2.0.0/gems/gym-1.9.0/lib/gym/commands_generator.rb:42:in `block (2 levels) in run'
    from /Library/Ruby/Gems/2.0.0/gems/commander-4.4.0/lib/commander/command.rb:178:in `call'
    from /Library/Ruby/Gems/2.0.0/gems/commander-4.4.0/lib/commander/command.rb:178:in `call'
    from /Library/Ruby/Gems/2.0.0/gems/commander-4.4.0/lib/commander/command.rb:153:in `run'
    from /Library/Ruby/Gems/2.0.0/gems/commander-4.4.0/lib/commander/runner.rb:444:in `run_active_command'
    from /Library/Ruby/Gems/2.0.0/gems/fastlane_core-0.52.0/lib/fastlane_core/ui/fastlane_runner.rb:36:in `run!'
    from /Library/Ruby/Gems/2.0.0/gems/commander-4.4.0/lib/commander/delegates.rb:15:in `run!'
    from /Library/Ruby/Gems/2.0.0/gems/gym-1.9.0/lib/gym/commands_generator.rb:61:in `run'
    from /Library/Ruby/Gems/2.0.0/gems/gym-1.9.0/lib/gym/commands_generator.rb:15:in `start'
    from /Library/Ruby/Gems/2.0.0/gems/gym-1.9.0/bin/gym:6:in `<top (required)>'
    from /usr/local/bin/gym:23:in `load'
    from /usr/local/bin/gym:23:in `<main>'
```

**Comment by jalmaas** [comment]
Seems link the path is set wrong. Double slash after "Developer". I have the same problem

**Comment by TKBurner** [comment]
@jalmaas @photoplyr So sorry for the delay on getting back to you. If you run 'sudo gem update fastlane' do you still see this issue. We've made a release since your comments and I want to make sure we test on the most recent version. Also can you share your full input and output (not just the crash) I think that'll help investigating.

**Comment by TKBurner** [comment]
@photoplyr  We are working on implementing a fix! Check out #6018 to follow along.
